src/backend/bsd.js

```diff
@@ -5,6 +5,7 @@ import querystring from 'querystring';
 import {parseString} from 'xml2js';
 import Promise from 'bluebird';
 import qs from 'querystring';
+import GCAudit from './data/models/gc-audit';
```

Comment:
Would actually make it BSDAudit.  The convention I'm following is BSDBlah is stuff related to BSD, GCBSDBlah are extended models (so GCBSDGroup has an underlying BSDGroup).  We may one day have a VANAudit as well.